src/qilib/utils/storage/interface.py

```diff
@@ -34,6 +34,10 @@ class NodeAlreadyExistsError(Exception):
     pass
 
 
+class StorageTimeoutError(Exception):
+    """ Raised when connection to storage can not be established."""
+
```

Comment:
For consistency maybe remove pass from other exceptions also. If you want you can also remove List from line 24 .

Reply:
done and done